doc/howto/gh-actions-badges.md

```diff
@@ -0,0 +1,84 @@
+# Use GitHub Actions To Place A MyBinder.org Badge On Pull Requests
+
+You can facilitate review of GitHub pull requests on [mybinder.org](https://mybinder.org) by using [GitHub Actions](https://github.com/features/actions) to place a ![Binder](https://mybinder.org/badge_logo.svg) badge or link on Pull Requests.  Below are several examples of how to accomplish this.
+
+To enable GitHub Actions, you must place .yaml files that define your GitHub Action workflow in the `/.github/workflows/` directory in your repository.
+
+## Example 1: Comment on pull request with a binder badge
+
+_Note: this example will not work on pull requests from forks.  This is to protect repositories from malicious actors.  To trigger a GitHub Action with sufficient permissions to comment on a pull request from a fork, you must trigger the action via another event, such as a comment or a label.  This is demonstrated in Example 2 below._
```

Comment:
I think a note like this could be made with a syntax like

````
```{note} optional note title here, default title is Note
Note body here
```
````

For more information see, https://myst-parser.readthedocs.io/en/latest/using/syntax.html#how-directives-parse-content. 

perhaps this?

````suggestion
```{note}
This example will not work on pull requests from forks.  This is to protect repositories from malicious actors.  To trigger a GitHub Action with sufficient permissions to comment on a pull request from a fork, you must trigger the action via another event, such as a comment or a label.  This is demonstrated in Example 2 below.
```
````

Reply:
oops, looks like we both had the same suggestion but you decided to be more helpful than me and actually write out the solution :-)

Reply:
btw - I don't believe you can over-ride `note` titles with the `note` directive (annoyingly). The directive will treat the first argument as *content*, not a title.

But, you can do this:

````
```{admonition} My title
:class: note

My note content
```
````

Reply:
Reading this i asked myself if this restriction was something enforced by github or by this job definition, im still not confident on this detail in general.

Reply:
it would be awesome if you could somehow automate the process of "only trigger this action on a PR from a fork if `/binder` is added by a member of the GitHub organization that owns the `main` repo.

Reply:
maybe @hamelsmu knows if that's possible?

Reply:
Yeah its possible, would take some more hacking though let me work on this